R/utils.R

```diff
@@ -121,3 +121,7 @@ new_opts <- function(x, opts, subclass = NULL, arg = NULL) {
     class = c(subclass, "vctrs_opts")
   )
 }
+
+glue_c <- function(..., env = caller_env()) {
```

Comment:
What's the point of this function?

Reply:
That's glue + c, to create named vectors with glue interpolation.

Reply:
My sense is that this should probably not yet be a function (just inline into `cnd_body.vctrs_error_recycle_incompatible_size`) or should be more specific: `glue_data_bullets()`. But I don't care that much.

Reply:
I like it, this is now `glue_error_bullets()`